README.Rmd

```diff
@@ -114,15 +114,12 @@ starwars %>%
 you control what goes in the nested column. `Z` is `N` but `r emo::ji("arrow_heading_down")`. 
 
 ```{r}
-tbl <- tibble(cyl = c(4, 6, 8), mpg = c(30, 25, 20)) 
+tbl <- tibble(cyl_threshold = c(4, 6, 8), mpg_threshold = c(30, 25, 20)) 
 tbl %>%
-  zest_join(mtcars, data = ~cyl == !!cyl & mpg < !!mpg)
+  zest_join(mtcars, data = ~cyl == !!cyl_threshold & mpg < !!mpg_threshold)
 ```
 
 In the rhs of the formula : 
 
   - `cyl` and `mpg` refer to columns of `mtcars`
-  - `!!cyl` and `!!mpg` refer to the current value from `tbl` 
-
-
-
+  - `!!cyl_threshold` and `!!mpg_threshold` refer to the current value from `tbl` 
```

Comment:
Maybe something like: 

- `cyl_threshold` and `mpg_threshold` refer to the current value from `tbl` because these columns don't exist in `mtcars`. If you wanted to refer to columns that are present both in `mtcars` and `tbl` you would have to unquote the columns in `tbl` with the unquoting operator, e.g. `!!cyl`